data/110/873/277/3/1108732773.geojson

```diff
@@ -5,16 +5,16 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.035035,
-    "geom:area_square_m":306664421.789024,
+    "geom:area_square_m":306664275.31873,
     "geom:bbox":"40.893594,44.776366,41.265029,45.064658",
     "geom:latitude":44.936456,
     "geom:longitude":41.110437,
     "iso:country":"RU",
     "label:eng_x_preferred_longname":[
-        "Armavir District"
+        "Armavir Gorsovet"
```

Comment:
Diving into a couple examples, I see this place referred to as the City of Armavir - an administrative unit with the status equal to that of the districts. 

https://en.wikipedia.org/wiki/Armavir,_Russia

Just curious, where is the "Gorsovet" coming from?

Reply:
I had taken these new values from Statoids. But it looks like they are too coarse compared to other options in `meso:*`...

Reply:
The only other widespread placetype label change was in Sudan which still seems appropriate (as long as the placetype label variant is also set).

Reply:
Reset back to `District` via https://github.com/whosonfirst-data/whosonfirst-data-admin-ru/pull/24/commits/66601ef140b14b95d1c32a7684b487fb2feee829.

Reply:
Thanks for the detail, super interesting! These changes all check out...